src/mapsector.cpp

```diff
@@ -109,6 +109,12 @@ void MapSector::insertBlock(MapBlock *block)
 }
 
 void MapSector::deleteBlock(MapBlock *block)
+{
+	removeBlock(block);
+	delete block;
```

Comment:
i clearly don't like the approach, remove/delete are synonyms it's totally confusing

Reply:
I'd suggest `detachBlock` and `deleteDetachedBlocks` is it makes sense and in lack of better proposals.

Reply:
I like that name. Thanks, done.